packages/code-du-travail-frontend/pages/_app.js

```diff
@@ -16,6 +16,10 @@ import { A11y } from "../src/a11y";
 // eslint-disable-next-line import/no-extraneous-dependencies
 import "katex/dist/katex.min.css";
 
+// Get tooltips web-component
+import("../web-components/tooltip");
```

Comment:
should we import that only in answer since it's where we have tooltip

Reply:
je croyais qu'on ne pouvait faire les imports que dans `_app.js` ? C'est ce que vous m'avez dit à la pr précédente concernenant le `import "katex/dist/katex.min.css";` ?

Reply:
oui, pour les css

Reply:
Yes jem'en occupe